Playbooks/playbook-Prisma_Cloud_Remediation_-_AWS_IAM_Password_Policy_Allows_Password_Reuse.yml

```diff
@@ -0,0 +1,410 @@
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "6"
+      "Yes":
+      - "4"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 480
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to: null
+      subject: null
+      body:
+        simple: ${incident.id} Autoremediate?
+      methods: []
+      format: ""
+      bcc: null
+      cc: null
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+      replyOptions:
+      - "Yes"
+      - "No"
+  "6":
+    id: "6"
+    taskid: 1214b667-e9c5-4efe-8503-d92b4f36cc0a
+    type: regular
+    task:
+      id: 1214b667-e9c5-4efe-8503-d92b4f36cc0a
+      version: -1
+      name: Manually update IAM password policy
+      description: |-
+        1. Login to the AWS console and navigate to the IAM dashboard
+        2. On the left navigation panel, click on Account Settings
+        3. Check Prevent password reuse and enter remember 5 passwords
+        4. Click on Apply password policy
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 810
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "7":
+    id: "7"
+    taskid: 83e1a4fe-23f8-4bfa-80ef-32dbeeea5c73
+    type: regular
+    task:
+      id: 83e1a4fe-23f8-4bfa-80ef-32dbeeea5c73
+      version: -1
+      name: Close investigation
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      assetid: {}
+      closeNotes: {}
+      closeReason: {}
+      emailclassification: {}
+      id:
+        complex:
+          root: incident
+          accessor: id
+      phishingsubtype: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 370,
+          "y": 980
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "8":
+    id: "8"
+    taskid: a8d06dc1-f186-4459-87b6-3d05d85f70d0
+    type: title
+    task:
+      id: a8d06dc1-f186-4459-87b6-3d05d85f70d0
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 690,
+          "y": 1330
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "9":
+    id: "9"
+    taskid: 23b39cf4-f5a4-4e88-8e71-b35011c009e4
+    type: regular
+    task:
+      id: 23b39cf4-f5a4-4e88-8e71-b35011c009e4
+      version: -1
+      name: Dismiss Prisma Cloud alert
```

Comment:
This command must task policy IDs or alert IDs as inputs, or it will fail the playbook.
So for these inputs we must have to get these two from the fetch data (I assume it's there), and have the `Is not empty` filter - relevant for all playbooks, because we want to dismiss the current alert